src/vpn/handler_setDns.go

```diff
@@ -15,7 +19,7 @@ import (
 
 var UnknownDnsManagementErr = errors.New("unknown dns management")
 
-func (h *Handler) setDns(dnsIp net.IP, clientIp net.IP, vpnNetwork net.IPNet, dnsManagement localDnsManagement) error {
+func (h *Handler) setDns(dnsIp net.IP, clientIp net.IP, vpnNetwork net.IPNet, dnsManagement dns.LocalDnsManagement) error {
```

Comment:
don't you want to move it into dns package. like detectDns

Reply:
sure